docs/api.rst

```diff
@@ -10,6 +10,11 @@ CellTree2d
 Changelog
 =========
 
+[Unreleased]
+------------
+- Add ``EdgeCellTree2d`` class to support 2D queries on 1D grids.
```

Comment:
I wouldn't use the term "1D grid", because it's clearly 2D data. I'd suggest "1D network" or something -- it's what the UGRID conventions mention. It'll work for line strings and such as well, so I'd use the term "linear geometry" or "linear features".

Reply:
Done!